Apps/APN/ios/AppDelegate.mm

```diff
@@ -155,7 +155,7 @@ - (void)application:(UIApplication *)application didRegisterForRemoteNotificatio
 {
   [pnHandlerObj application:application deviceToken:deviceToken];
  [RNCPushNotificationIOS didRegisterForRemoteNotificationsWithDeviceToken:deviceToken];
-  [super application:application didRegisterForRemoteNotificationsWithDeviceToken:deviceToken];
+//  [super application:application didRegisterForRemoteNotificationsWithDeviceToken:deviceToken];
```

Comment:
I couldn't get the APN sample app to compile without commenting out these lines. Not sure why?